### ISSUE TITLE
document XMesh communication

### ISSUE THREAD

**Issue by lahabana** [kumahq/kuma-website#817]
### Description

The feature is there we should document it

**Comment by lahabana** [comment]
@michaelbeaumont was this completed in #831 ?

**Comment by michaelbeaumont** [comment]
Yes, though there was one TODO from friday about making it clear how Gateways are accessed
